openmc/statepoint.py

```diff
@@ -13,17 +13,28 @@
 
 
 class SourceSite(object):
+    """A single source site, usually resulting from fission.
```

Comment:
Perhaps clarify this as fission or (n,xn) reactions? I prefer transparency / precise explanations over words like "usually", but that's just me. If going beyond "usually fission" is problematic, that's fine too.